### PR TITLE
feat:button-component

### PR DESCRIPTION
**Pull request by tanishk15sharma**
I have added My Button Component : 

1)Basic Buttons - having 3 types primary , secondary and link 
2)Stylish Buttons - having 3 types large , hover , loading

### CODE REVIEW

File: style/components/button.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+.btn {
+  padding: 0.8rem 1rem;
+  border: none;
+  cursor: pointer;
+  font-size: 18px;
```
Comment (JantuDeb) [inline style/components/button.css:5]: Use rem for `font-size`

File: style/components/button.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  color: white;
+  background-color: gray;
+  margin-left: 0.5rem;
+  margin-right: 0.5rem;
```
Comment (JantuDeb) [inline style/components/button.css]: You can use `margin: 0 0.5rem`

File: style/components/button.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  transition: 0.2s;
+}
+
+.primary {
```
Comment (JantuDeb) [inline style/components/button.css]: You can add the prefix `btn` or` button` , `btn-primary` is clearer. You can use this convention for all the classes if you want to.

**Review by JantuDeb — APPROVED** [review]
I have commented out the issue, you can fix and then merge

File: style/components/button.css (hunk 2 above)
Comment (tanishk15sharma) [reply style/components/button.css]: Okay I will do this

File: style/components/button.css (hunk 3 above)
Comment (tanishk15sharma) [reply style/components/button.css]: sure doing the required change

**Review by JantuDeb — APPROVED** [review]
This code is fine, you can merge it

**Comment by shaikhhamza11** [comment]
Looks good . Just a suggestion you can use rem units instead of pixels .

**Pull request merged**